Modules/Calcs.lua

```diff
@@ -106,23 +114,200 @@ function calcs.getMiscCalculator(build)
 	-- Run base calculation pass
 	local env = calcs.initEnv(build, "CALCULATOR")
 	calcs.perform(env)
+	local fullDPS = calcs.calcFullDPS(build, "CALCULATOR")
+	env.player.output.SkillDPS = fullDPS.skills
+	env.player.output.FullDPS = fullDPS.combinedDPS
 	local baseOutput = env.player.output
 
 	return function(override)
 		env = calcs.initEnv(build, "CALCULATOR", override)
 		calcs.perform(env)
+		GlobalCache.dontUseCache = true
```

Comment:
how big a hit is this ?  It seems like a big enough deal that I'd add a comment here for the bad things that happen when dontUseCache isn't set here

Reply:
It's there not as a performance measure, but to prevent the comparison over-ride using cached values so it can determine the benefit of adding a new node/item/gem into the build and seeing what the FullDPS gain is.

If it uses the cached values (which are stored a few lines above in 117-119) it would see no net benefit from any additions for the FullDPS roll-up comparison

Reply:
I'll add a comment though, no harm in that and it will explain it